android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DevCycleClient.kt

```diff
@@ -615,7 +615,7 @@ class DevCycleClient private constructor(
             return this
         }
 
-        fun withLogLevel(logLevel: LogLevel): DevCycleClientBuilder {
+        fun logLevel(logLevel: LogLevel): DevCycleClientBuilder {
```

Comment:
we have the `with` prefix for all other builder command, should keep it and add this as a new function

Reply:
if truly required, or keep using the with prefix function